src/renderers/shared/stack/reconciler/ReactUpdateQueue.js

```diff
@@ -58,15 +59,18 @@ function getInternalInstanceReadyForUpdate(publicInstance, callerName) {
   }
 
   if (__DEV__) {
-    warning(
-      ReactCurrentOwner.current == null,
-      '%s(...): Cannot update during an existing state transition (such as ' +
-      'within `render` or another component\'s constructor). Render methods ' +
-      'should be a pure function of props and state; constructor ' +
-      'side-effects are an anti-pattern, but can be moved to ' +
-      '`componentWillMount`.',
-      callerName
-    );
+    if (warnedOnce === false) {
+      warnedOnce = true;
+      warning(
+        ReactCurrentOwner.current == null,
```

Comment:
Yeah, it will. It's just a "sketch" now to show how I was thinking. Doing like a conditional like `if(!ReactCurrentOwner.current == null && warnedOnce == true)` might make tests fail, at least did when I did that